### ISSUE TITLE
First snap flow: "continue" button is confusing

### ISSUE THREAD

**Issue by nottrobin** [canonical-websites/snapcraft.io#1324]
When I first go to the Python first snap flow page at https://snapcraft.io/first-snap/python:

![image](https://user-images.githubusercontent.com/519935/48617026-309c2480-e98d-11e8-9503-1e82ad065731.png)

It seemed to me that Linux was already selected (as it was the only colourful logo), so I tried to click the green "continue" button. The button looks clickable, but it does nothing until I actually click on the "Linux" logo.

**Comment by lyubomir-popov** [comment]
I think the issue here is that the disabled logos do not look disabled enough. Could we have them in colour, but at 50% opacity?

**Comment by nottrobin** [comment]
@lyubomir-popov to me the primary problem is that the "Continue" button is not disabled, or not clearly disabled, when in fact clicking it does nothing.

**Comment by nottrobin** [comment]
Another suggestion: If, in fact, the only option available is Linux, might it not make sense for that to be selected automatically, and the form expanded, when you first visit the page?

**Comment by lyubomir-popov** [comment]
@nottrobin @therealjuan On second thought asking the question "What operating system are you using?" and then providing just one option feels like an unpleasant surprise if you're using any of the disabled options. I wonder if there's a more polite way of saying that the guide is linux only for the time being? Could we skip to the next step with a note on top saying "MacOS / Win guides coming soon"? This way, you land on the page, you immediately see whether this guide is relevant to you, rather than having to try and interact and get frustrated in the process.

**Comment by therealjuan** [comment]
- Continue should be disabled until you choose an OS 

- Visually these controls do NOT have enough contrast between states so it's difficult to appreciate their difference. I have some serious concerns about the accessibility of the different states of these controls

- Having only the Linux guide enabled was a last minute decision because there was a problem with the MacOS guide that we are working on fixing and will happen soon-ish

-  Pre-selecting could be interesting and perhaps we can read the browser' user agent to choose an OS depending on its value

**Comment by nottrobin** [comment]
> Pre-selecting could be interesting and perhaps we can read the browser' user agent to choose an OS depending on its value

Yes I think we should definitely do this. It's trivial to implement, and should align with what the user wants in the vast majority of cases.

**Comment by matthewpaulthomas** [comment]
If this has Low priority, it’s unlikely to be improved directly before macOS or Windows instructions are added.
